src/nuclio/functions/function-collapsing-row/function-collapsing-row.component.js

```diff
@@ -80,6 +82,7 @@
         ctrl.functionsService = FunctionsService;
         ctrl.getFunctionsTableColSize = TableSizeService.getFunctionsTableColSize;
         ctrl.isDemoMode = ConfigService.isDemoMode;
+        ctrl.lodash = lodash;
```

Comment:
Seems like this line is redundant.
I see no use of `$ctrl.lodash` in the corresponding HTML template ([src/nuclio/functions/function-collapsing-row/function-collapsing-row.tpl.html](https://github.com/iguazio/dashboard-controls/pull/1157/files#diff-ae4486ca037ebdd6bf86c5662924f3ecaf011fd8afe79564267c4288f7292a3e)).